plugins/quickcl.py

```diff
@@ -482,6 +469,22 @@ def _setup_sbar_cont(self):
         for sctrl in self._dlg.streambar_controller.stream_controllers:
             sctrl.stream_panel.show_remove_btn(False)
 
+        # Move to pass-through filter and provide a cl-filter control in the CL stream
+        main_data = self.main_app.main_data
+        if main_data.cl_filter and "band" in main_data.cl_filter.axes:
+            # find the "pass-through"
+            bdef = main_data.cl_filter.axes["band"]
+            for b, bn in bdef.choices.items():
+                if bn == "pass-through":
+                    main_data.cl_filter.moveAbs({"band": b})
+                    break
+            else:
+                logging.debug("Pass-through not found in the CL-filter")
+            for s_cont in self._dlg.streambar_controller.stream_controllers:
+                if isinstance(s_cont.stream, LiveCLStream):
+                    s_cont.add_axis_entry("band", main_data.cl_filter)
```

Comment:
I'd be nice to use the "stream_configs", as in `show_mn_axes()`. You'd need to use the CLSettingsStream. In particular, this renames the axis to "Filter" instead of "Band".